packages/compass-crud/src/components/editable-element-field.less

```diff
@@ -3,14 +3,12 @@
   border: none;
   padding-left: 1px;
   -webkit-user-select: text;
-  max-width: 15%;
-  text-overflow: ellipsis;
```

Comment:
Do we still want some ellipses when the field name and value are more than the width of the window?
<img width="772" alt="Screen Shot 2022-02-09 at 1 40 33 PM" src="https://user-images.githubusercontent.com/1791149/153268282-3e967d2a-3dc1-4790-acdc-b8b3a2722e61.png">

Reply:
Would increasing the `max-width` to something like `70%` and re-adding `text-overflow: ellipsis;` work?